README-WAF.md

```diff
@@ -62,7 +62,18 @@ waf list
 
 ## Program groups ##
 
-The programs in ardupilot are categorized into the following groups:
+Program groups are used to represent a class of programs. They can be used to
+build all programs of a certain class without having to specify each program. A
+program can belong to more than one group.
```

Comment:
See the "Main groups" section. A program will always belong to one of the main groups. And the main group defines where the program ends up.